app/components/icons/SoftwareEngineer.tsx

```diff
@@ -0,0 +1,213 @@
+import type { SVGProps } from "react";
```

Comment:
For this pr it's ok to leave this in but we should remove this when prismic is hooked up and the svg should just get pulled from prismic.